src/manager/manager.c

```diff
@@ -812,13 +811,37 @@ static int manager_method_metrics_disable(sd_bus_message *m, void *userdata, UNU
         return sd_bus_reply_method_return(m, "");
 }
 
+/*************************************************************************
+ ************** org.containers.hirte.Manager.SetLogLevel ***************
+ ************************************************************************/
+
+static int manager_method_set_log_level(sd_bus_message *m, UNUSED void *userdata, UNUSED sd_bus_error *ret_error) {
+        const char *level = NULL;
+
+        int r = sd_bus_message_read(m, "s", &level);
+        if (r < 0) {
+                hirte_log_errorf("Failed to read the parameter: %s", strerror(-r));
+                return sd_bus_reply_method_errorf(
+                                m, SD_BUS_ERROR_FAILED, "Failed to read the parameter: %s", strerror(-r));
```

Comment:
```suggestion
                return sd_bus_reply_method_errorf(m, SD_BUS_ERROR_INVALID_ARGS, "Invalid arguments");
```
@mwperina That would be gread, but as far as I know in case of multiple params we are not able to, unfortunately. Thats why we return just an `Invalid arguments` message. 
@ArtiomDivak The `hirte_log_errorf` isn't necessary here since its a "user related" error.